cmd/mapstructure-to-hcl2/mapstructure-to-hcl2.go

```diff
@@ -575,8 +575,8 @@ func ToSnakeCase(str string) string {
 	return strings.ToLower(snake)
 }
 
-func goFmt(b []byte) []byte {
-	fb, err := format.Source(b)
+func goFmt(filename string, b []byte) []byte {
+	fb, err := imports.Process(filename, b, nil)
```

Comment:
Will imports also do the same things as format ? Shouldn't we run both ?

Reply:
Yes imports will do the same things as format, as per the pkg documentation:
```
In addition to fixing imports, goimports also formats your code in the same style as gofmt so it can be used as a replacement for your editor's gofmt-on-save hook.
```

After re-running `make generate` there was only one change so it looks to be true.

Reply:
Ah bummer I didn't see that line ! Nice !